site/public/acmApp/components/officers/officersController.js

```diff
@@ -0,0 +1,33 @@
+var logger = console;  //someday, a different logger..
+
+angular
+  .module('acmApp')
+  .controller('officersController', ['officersService', officersController]);
+
+function officersController(officersService) {
+
+  vm = this;
```

Comment:
I've been using `var self = this` at work so that it can be consistent across all controllers. Should we do something like that?